dot/peerset/test_helpers.go

```diff
@@ -25,22 +25,28 @@ const (
 	peer2         = peer.ID("testPeer2")
 )
 
-func newTestPeerSet(t *testing.T, in, out uint32, bootNodes, reservedPeers []peer.ID, reservedOnly bool) *Handler {
+const allocTimeDuration = 2 * time.Second
+
+//go:generate mockgen -destination=mock_message_processor_test.go -package $GOPACKAGE . MessageProcessor
+func newTestPeerSet(t *testing.T, maxIn, maxOut uint32, bootNodes,
```

Comment:
nit add an empty line so it doesn't show as a comment of the function when hovering over it in an IDE

```suggestion

func newTestPeerSet(t *testing.T, maxIn, maxOut uint32, bootNodes,
```